utils.py

```diff
@@ -276,6 +282,11 @@ def get_search_props():
         if not hasattr(wm, "blenderkit_brush"):
             return
         props = wm.blenderkit_brush
+
+    if uiprops.asset_type == "NODEGROUP":
+        if not hasattr(wm, "blenderkit_nodegroup"):
```

Comment:
👍

Reply:
I didn't get this one, I guess that mean nothing to fix? There could be a dict that could replace those lines of code.